app/views/parent/kids/index.html.erb

```diff
@@ -1,19 +0,0 @@
-<h1> All My Kids </h1>
-
-<table>
-  <tr>
-    <th> First Name </th>
-    <th> Last Name </th>
-    <th> Birthdate </th>
-    <th> SIN </th>
-  </tr>
-  <% @kids.each do |kid| %>
-    <tr>
-      <td> <%= kid.first_name %> </td>
-      <td> <%= kid.last_name %> </td>
-      <td> <%= kid.birthdate %> </td>
-      <td> <%= kid.sin %> </td>
-      <td> <%= link_to "Edit Kid", edit_parent_kid_path(:id => kid.id) %> </td>
-    </tr>
-  <% end %>
-</table>
```

Comment:
Please see https://github.com/DerangedMind/fluffy-camp-fire/pull/33

We decided to scrap it and just show all the info on the profile page.